### PR TITLE
OSDOCS#7303: Update release notes for the cert-manager Operator 1.11

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-7303

Link to docs preview: https://63190--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-notable-technical-changes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc
Comment (xingxingxia) [inline release_notes/ocp-4-14-release-notes.adoc:247]:
```suggestion
[id="ocp-4-14-cert-manager-operator-1-11"]
```

**Comment by xenolinux** [comment]
@xingxingxia Addressed the suggestion. Can you PTAL?

**Review by skopacz1 — APPROVED** [review]
LGTM. Please wait until @adellape approves this review before proceeding, thank you.

**Review by adellape — APPROVED** [review]
@skopacz1 @xenolinux LGTM 👍

**Pull request merged**
